systems/crusher_quick_start_guide.rst

```diff
@@ -1340,33 +1340,52 @@ For example:
 Theoretical Roofline
 ^^^^^^^^^^^^^^^^^^^^
 
-The theoretical (attainable) roofline constructs a theoretical maximum performance for each operational intensity.
+The theoretical (this is NOT attainable FLOPS/s) roofline constructs a theoretical maximum performance for each operational intensity.
```

Comment:
Also, instead of "constructs a theoretical maximum performance for each operational intensity", I think I would say "constructs a plot of theoretical maximum performance versus operational intensity"

Reply:
I don't know that `peak` is the differentiating word, is it? My understanding is that we have a `theoretical peak` and an `attainable peak`, so grammatically, it may look like "The theoretical-peak (not attainable) roofline ..." And maybe a note section immediately after that explains `theoretical` vs `attainable` vs `achieved` terms?

Reply:
Good point. I was trying to be aware of not mixing the terms and I still used an ambiguous term; shows how much definitions are needed : )

But yes, I agree with your suggestion.